harness/tests/experiment/pytorch/test_pytorch_trial.py

```diff
@@ -136,7 +136,7 @@ def test_checkpointing_and_restoring(self, tmp_path: pathlib.Path) -> None:
         self.checkpoint_and_restore(updated_hparams, tmp_path, (100, 100))
 
     @pytest.mark.skipif(not torch.cuda.is_available(), reason="no gpu available")
-    @pytest.mark.gpu
+    # @pytest.mark.gpu
```

Comment:
what's going on here?

Reply:
We have some GPU tests that were not maintained after changes to our trial context/controller objects. I created a ticket for these, MLG-479. I figure enabling these is out of the scope of this PR, so I kept them disabled. If we decide that these can replace some e2e_tests, then it may be worth fixing.